### PR TITLE
Rde medicationrequest

### PR DESCRIPTION
**Pull request by klwhaley**
Change to make RDE_O11/RDE_O25 only use RXE for medication request

### CODE REVIEW

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 1]
```diff
@@ -1053,6 +1050,42 @@ public void medicationRequestIdentifierTest() {
         assertThat(system).isNull();
         type = identifier.getType();
         DatatypeUtils.checkCommonCodeableConceptAssertions(type, "VN", "Visit number", "http://terminology.hl7.org/CodeSystem/v2-0203", null);
+        
+        // Identifier 2: extID based on RXE-1.1 and RXE-1.3
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1054]: This is based on RXE-2, not RXE-1

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 2]
```diff
@@ -1053,6 +1050,42 @@ public void medicationRequestIdentifierTest() {
...
+        identifier = medReq.getIdentifier().get(1);
+        value = identifier.getValue();
+        system = identifier.getSystem();
+        assertThat(value).isEqualTo("999-NDC"); // RXE-1.1 and RXE-1.3
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1058]: This is based on RXE-2, not RXE-1

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 3]
```diff
@@ -1053,6 +1050,42 @@ public void medicationRequestIdentifierTest() {
...
+        // Identifier 1: Visit number
+        identifier = medReq.getIdentifier().get(0);
+        value = identifier.getValue();
+        system = identifier.getSystem();
+        assertThat(value).isEqualTo("789789"); // PV1.19
+        assertThat(system).isNull();
+        type = identifier.getType();
+        DatatypeUtils.checkCommonCodeableConceptAssertions(type, "VN", "Visit number", "http://terminology.hl7.org/CodeSystem/v2-0203", null);
+
+        // Identifier 2: extID based on RXO-1.1 and RX-O1.3
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1082]: This is based on RXE-2, not RXE-1

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 4]
```diff
@@ -1053,6 +1050,42 @@ public void medicationRequestIdentifierTest() {
...
+        identifier = medReq.getIdentifier().get(1);
+        value = identifier.getValue();
+        system = identifier.getSystem();
+        assertThat(value).isEqualTo("999-NDC"); // RXO-1.1 and RXO-1.3
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1086]: This is based on RXE-2, not RXE-1

**Review by LisaWellman — APPROVED** [review]
Approving with minor comments

**Review by xpander53 — APPROVED** [review]

File: src/main/resources/hl7/message/OMP_O09.yml [hunk 5]
```diff
@@ -45,6 +45,8 @@ resources:
       - .ORC
       - .COMPONENT.RXC
       - MSH
+      - PATIENT.PATIENT_VISIT.PV1
+      - PATIENT.PID
```
Comment (xpander53) [inline src/main/resources/hl7/message/OMP_O09.yml:49]: I'm curious why you removed these

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 6]
```diff
@@ -1025,11 +1022,11 @@ public void medicationRequestIdentifierTest() {
         type = identifier.getType();
         DatatypeUtils.checkCommonCodeableConceptAssertions(type, "VN", "Visit number", "http://terminology.hl7.org/CodeSystem/v2-0203", null);
 
-        // Identifier 2: extID based on RXO-1.2
```
[the remark this answers is not in the document]
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1028]: fixed

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 7]
```diff
@@ -1025,11 +1022,11 @@ public void medicationRequestIdentifierTest() {
...
+        // Identifier 2: extID based on RXO1.1
         identifier = medReq.getIdentifier().get(1);
         value = identifier.getValue();
         system = identifier.getSystem();
-        assertThat(value).isEqualTo("DOCUSATE SODIUM 100 MG CAPSULE"); // RXO1.2
+        assertThat(value).isEqualTo("DOCUSATE SODIUM 100 MG CAPSULE"); // RXO1.1
```
[the remark this answers is not in the document]
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1029]: fixed

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java [hunk 8]
```diff
@@ -1043,7 +1040,7 @@ public void medicationRequestIdentifierTest() {
 
         // Expect 1 identifier
```
[the remark this answers is not in the document]
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1041]: fixed

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java (hunk 2 above)
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1058]: fixed

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java (hunk 1 above)
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1054]: fixed

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java (hunk 3 above)
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1082]: fixed

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java (hunk 4 above)
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7IdentifierFHIRConversionTest.java:1086]: fixed

File: src/main/resources/hl7/message/OMP_O09.yml (hunk 5 above)
Comment (klwhaley) [reply src/main/resources/hl7/message/OMP_O09.yml:49]: i didn't remove them, I added them OMP Medication request should have PV1/PID available for identifiers if they are there

**Pull request merged**